src/pe_reports/stylesheet.py

```diff
@@ -5,96 +5,91 @@
 from pptx.enum.chart import XL_LEGEND_POSITION
 from pptx.util import Pt
 
+# Font Attributes
+SMALL = Pt(10)
+LARGE = Pt(28)
+BLUE = RGBColor(3, 37, 126)
+GREEN = RGBColor(20, 200, 100)
+WHITE = RGBColor(255, 255, 255)
+
 
 class Paragraph:
     """Simple class to call text frame attributes."""
 
-    def shapes(self, slide):
+    @staticmethod
+    def shapes(slide):
         """Create a text frame."""
         for shape in slide.shapes:
             if not shape.has_text_frame:
                 continue
         return shape
 
-    def shapes_find(self, slide):
+    @staticmethod
+    def shapes_find(slide):
         """Find text frames."""
         for shape in slide.shapes:
             print(shape.name)
-        return
+        return shape
 
-    def text_frame(self, shape):
+    @staticmethod
+    def text_frame(shape):
         """Clear the content of a text frame."""
         text_frame = shape.text_frame
         text_frame.clear()
         return text_frame
 
-    def text_frame_ov_val(self, slide, shape, name):
+    @staticmethod
+    def text_frame_ov_val(slide, name):
         """Text style for overview page values."""
         for shape in slide.shapes:
             if shape.name == name:
                 text_frame = shape.text_frame
                 text_frame.clear()
         return text_frame
 
-    def text_frame_key_metric(self, slide, shape, name):
+    @staticmethod
+    def text_frame_key_metric(slide, name):
         """Text style for metrics."""
         for shape in slide.shapes:
             if shape.name == name:
                 text_frame = shape.text_frame
                 text_frame.clear()
         return text_frame
```

Comment:
You need to define this variable in case there is no `shape` in `slide.shapes` such that `shape.name == name`.

Reply:
Yes creating an issue and tying it to the work in #14 is reasonable.

Reply:
Resolved in d6bab3f